subsys/bluetooth/audio/pacs.c

```diff
@@ -202,15 +202,46 @@ static void available_context_cfg_changed(const struct bt_gatt_attr *attr, uint1
 	LOG_DBG("attr %p value 0x%04x", attr, value);
 }
 
+static enum bt_audio_context pacs_get_available_contexts_for_conn(struct bt_conn *conn,
```

Comment:
should this have been prefixed with `bt_`?

Reply:
Hmm, usually I don't prefix `static` functions with `bt_`, but that's not a rule we have.

Reply:
I missed it was static. My eyes saw the one in the header, which is not part of the commit.